system/Commands/Database/CreateSeeder.php

```diff
@@ -0,0 +1,174 @@
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @package    CodeIgniter
+ * @author     CodeIgniter Dev Team
+ * @copyright  2019-2020 CodeIgniter Foundation
+ * @license    https://opensource.org/licenses/MIT	MIT License
+ * @link       https://codeigniter.com
+ * @since      Version 4.0.0
+ * @filesource
+ */
+
+namespace CodeIgniter\Commands\Database;
+
+use CodeIgniter\CLI\BaseCommand;
+use CodeIgniter\CLI\CLI;
+use Config\Services;
+
+/**
+ * Creates a new seeder file.
+ *
+ * @package CodeIgniter\Commands
+ */
+class CreateSeeder extends BaseCommand
+{
+	/**
+	 * The group the command is lumped under
+	 * when listing commands.
+	 *
+	 * @var string
+	 */
+	protected $group = 'Database';
+
+	/**
+	 * The Command's name
+	 *
+	 * @var string
+	 */
+	protected $name = 'db:seed:create';
+
+	/**
+	 * the Command's short description
+	 *
+	 * @var string
+	 */
+	protected $description = 'Creates a new seeder file.';
+
+	/**
+	 * the Command's usage
+	 *
+	 * @var string
+	 */
+	protected $usage = 'db:seed:create [seeder_name] [options]';
+
+	/**
+	 * the Command's Arguments
+	 *
+	 * @var array
+	 */
+	protected $arguments = [
+		'seeder_name' => 'The seeder file name',
+	];
+
+	/**
+	 * the Command's Options
+	 *
+	 * @var array
+	 */
+	protected $options = [
+		'-n' => 'Set seeder namespace',
+	];
+
+	/**
+	 * Creates a new migration file with the current timestamp.
+	 *
+	 * @param array $params
+	 */
+	public function run(array $params = [])
+	{
+		helper('inflector');
+		
+		$name = array_shift($params);
+
+		if (empty($name))
+		{
+			$name = CLI::prompt(lang('Seed.nameFile'));
+		}
+
+		if (empty($name))
+		{
+			CLI::error(lang('Seed.badName'));
+			return;
+		}
+
+		$ns       = $params['-n'] ?? CLI::getOption('n');
+		$homepath = APPPATH;
+
+		if (! empty($ns))
+		{
+			// Get all namespaces
+			$namespaces = Services::autoloader()->getNamespace();
+
+			foreach ($namespaces as $namespace => $path)
+			{
+				if ($namespace === $ns)
+				{
+					$homepath = realpath(reset($path));
+					break;
+				}
+			}
+		}
+		else
+		{
+			$ns = 'App';
+		}
+
+		// full path
+		$path = $homepath . '/Database/Seeds/' . $name . '.php';
```

Comment:
And the beginning forward slash in `/Database/Seeds/` needs to be removed.

Reply:
It shows the full path to the file
![Screenshot from 2020-07-06 12-42-32](https://user-images.githubusercontent.com/15793304/86628084-97b58500-bf86-11ea-95a0-3bc5d1b774cd.png)

Reply:
Yes. That's because of the `str_replace` at the last line. But if you'll `var_dump` the `$path` assuming we don't supply the namespace option, we will get this.
![image](https://user-images.githubusercontent.com/51850998/86755009-e42abe80-c073-11ea-9ac4-2289b9d011ea.png)

Notice the `app\/Database`.